src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styles from './Pagination.module.sass'
+import { useDispatch, useSelector } from 'react-redux'
+import { setFilter, setCurrentPage } from '../../store/slices/ProductSlice'
+import { useTranslation } from 'react-i18next';
+
+function Pagination() {
+
+    const { t } = useTranslation()
+
+    const dispatch = useDispatch()
+
+    const pagination = useSelector( state => state.products.pagination)
+    const filter = useSelector( state => state.products.filter)
```

Comment:
```suggestion
    const { t } = useTranslation()
    const dispatch = useDispatch()
    const pagination = useSelector(state => state.products.pagination)
    const filter = useSelector(state => state.products.filter)
```
Đâu đã thấy sửa đâu nhỉ? Với lại sửa format code cho dễ nhìn 1 chút, để khoảng trắng như này bị dư thừa đó